OlinkAnalyze/vignettes/plate_randomizer.Rmd

```diff
@@ -0,0 +1,141 @@
+
+
+```{r subjects_randomization, message=FALSE,results='hide'}
+randomized.manifest_b <- olink_plate_randomizer(manifest,SubjectColumn="SubjectID", 
+                                                available.spots=c(78,30,30), 
+                                                iterations = 500,
+                                                seed=123456)
+```
+
+The output of the randomization function would be a "tibble" including SampleID, SubjectID etc. assigned to well positions. Columns include same columns as Manifest with additional columns:
+
+  - plate: Plate number
+
+  - column: Column on the plate
+
+  - row: Row on the plate
+
+  - well: Well location on the plate
+
+## Visualization
+
+To illustrate the goodness of randomization, both `olink_displayPlateLayout()` and `olink_displayPlateDistributions` functions could be used.
+
+### Plate layouts
+
+The `olink_displayPlateLayout()` function could be used to visualize the layouts of the plate and specifying the color for the variable of interest using the `color_g` argument. The label of the colored variable could be shown in the plot via the `include.label` argument.
+
+```{r, fig.height = 7, fig.width = 7, fig.align = "center"}
+olink_displayPlateLayout(randomized.manifest_b, fill.color = 'SubjectID', include.label = FALSE)
+```
+
+The label of the colored variable could be shown in the plot via the `include.label` argument.
+
+```{r,fig.height = 7, fig.width = 7, fig.align = "center"}
+olink_displayPlateLayout(randomized.manifest_b, fill.color = 'SubjectID', include.label = TRUE)
+```
+
+### Plate distribution
+
+The distribution of the given grouping variable on each plate could be visualized via a bar chart using function `olink_displayPlateDistributions`. By setting `fill.color = 'SubjectID'`, we could make sure all the samples from the same subject were put on the same plate. 
+
+```{r,fig.height = 7, fig.width = 7, fig.align = "center"}
+olink_displayPlateDistributions(randomized.manifest_b, fill.color = 'SubjectID')
+```
+
+We could also check the distribution of other variables. For examples, the distribution of the variable `Site` could be visualized by setting `fill.color = 'Site'`. 
+
+```{r,fig.height = 7, fig.width = 7, fig.align = "center"}
+olink_displayPlateDistributions(randomized.manifest_b, fill.color = 'Site')
+```
+
+In the case of the randomization is not performed well, the randomization could be performed again with different seed number and iterations number. 
```

Comment:
Well this is not true right? Since plate 2 and 3 is only 30 sampels each, if plate 2 is ~45% = 14 samples, and plate 3 is ~60% = 18 samples. Expecting 15 -15 is probably not reasonable IMO. 
Or change eeverything so instead of 88 -30-30 you have like 50-50-48